tests/rpc/conftest.py

```diff
@@ -0,0 +1,26 @@
+import asyncio
+import os
+import pytest
+import tempfile
+from threading import Thread
+import uuid
+
+from evm.rpc.ipc import (
+    start,
+)
+
+
+@pytest.fixture(scope='session')
+def ipc_pipe():
+    tmpdir = tempfile.gettempdir()
+    return os.path.join(tmpdir, 'test-%s.ipc' % uuid.uuid4())
```

Comment:
What do you think about changing this to a `yield` and doing an explicit cleanup check to be sure we're not leaving this file laying around?  Is that even a possible issue?  If not, please ignore.